### PR TITLE
[castai-cluster-controller] Granted all permissions within castai namespace

### PR DESCRIPTION
**Pull request by andrejatcastai**

### CODE REVIEW

File: charts/castai-cluster-controller/Chart.yaml [hunk 1]
```diff
@@ -2,5 +2,5 @@ apiVersion: v2
 name: castai-cluster-controller
 description: CAST AI cluster controller deployment chart.
 type: application
-version: 0.12.0
+version: 0.13.0
```
Comment (varnastadeus) [inline charts/castai-cluster-controller/Chart.yaml:5]: Sorry, 0.13.0 is already deployed 

```suggestion
version: 0.14.0
```

File: charts/castai-cluster-controller/Chart.yaml [hunk 2]
```diff
@@ -2,5 +2,5 @@ apiVersion: v2
 name: castai-cluster-controller
 description: CAST AI cluster controller deployment chart.
 type: application
-version: 0.13.0
+version: 0.14.0
 appVersion: "v0.10.1"
```
Comment (varnastadeus) [inline charts/castai-cluster-controller/Chart.yaml:6]: as you have changed cluster-controller, I would suggest to update appVersion in one go when it is released

File: charts/castai-cluster-controller/templates/rbac.yaml [hunk 3]
```diff
@@ -73,19 +73,10 @@ metadata:
...
-  - apiGroups: [""]
-    resources: [ "serviceaccounts", "secrets", "configmaps" ]
-    verbs: [ "get", "list", "create", "update", "delete", "patch" ]
-  - apiGroups: [ "rbac.authorization.k8s.io" ]
-    resources: [ "roles" ]
-    verbs: [ "get", "list", "create", "update", "delete", "patch" ]
-  - apiGroups: [ "rbac.authorization.k8s.io" ]
-    resources: [ "rolebindings" ]
-    verbs: [ "get", "list", "patch", "update", "delete" ]
+  # controller can do anything in castai-agent namespace; this is required to fully manage castai components
```
Comment (anjmao) [inline charts/castai-cluster-controller/templates/rbac.yaml:76]: Need to test if this works with auto update.

File: charts/castai-cluster-controller/Chart.yaml (hunk 2 above)
Comment (andrejatcastai) [reply charts/castai-cluster-controller/Chart.yaml:6]: Got it, will changed app version here once it is released

**Review by anjmao — APPROVED** [review]

**Pull request merged**